packages/tooltip/src/Tooltip.spec.tsx

```diff
@@ -361,27 +361,53 @@ describe('packages/tooltip', () => {
         expect(button).toHaveTextContent(buttonText);
       });
 
-      test('when trigger event is click, `click` triggers opening and closing of tooltip', async () => {
-        const { button, getByTestId } = renderTrigger({
-          triggerEvent: 'click',
+      describe('triggerEvent is `click`', () => {
+        test('click event triggers opening and closing of tooltip', () => {
+          const { button, getByTestId } = renderTrigger({
+            triggerEvent: 'click',
+          });
+          userEvent.click(button);
+          const tooltip = getByTestId(tooltipTestId);
+          expect(tooltip).toBeInTheDocument();
+          userEvent.click(button);
+          waitForElementToBeRemoved(tooltip);
+        });
+
+        test('clicking the backdrop closes the tooltip', () => {
+          const { button, getByTestId } = renderTrigger({
+            triggerEvent: 'click',
+          });
+          userEvent.click(button);
+          const tooltip = getByTestId(tooltipTestId);
+          const backdrop = getByTestId('backdrop');
+          userEvent.click(backdrop);
+          waitForElementToBeRemoved(tooltip);
         });
-        userEvent.click(button);
-        const tooltip = getByTestId(tooltipTestId);
-        expect(tooltip).toBeInTheDocument();
-        userEvent.click(button);
-        await waitForElementToBeRemoved(tooltip);
       });
 
-      test('when trigger event is hover, `hover` triggers opening and closing of tooltip', () => {
-        const { button, getByTestId } = renderTrigger({
-          triggerEvent: 'hover',
+      describe('triggerEvent is `hover`', () => {
+        test('hover event triggers opening and closing of tooltip', () => {
+          const { button, getByTestId } = renderTrigger({
+            triggerEvent: 'hover',
+          });
+          userEvent.hover(button);
+          waitFor(() => {
+            const tooltip = getByTestId(tooltipTestId);
+            expect(tooltip).toBeInTheDocument();
+            userEvent.unhover(button);
+            waitForElementToBeRemoved(tooltip);
+          });
         });
-        userEvent.hover(button);
-        waitFor(async () => {
+
+        test('clicking the backdrop does not close the tooltip', () => {
+          const { button, getByTestId } = renderTrigger({
+            triggerEvent: 'click',
```

Comment:
Should this be `hover`, its under the `hover` test?

Reply:
Ha yeah 🙃